src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -628,11 +632,10 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.WAVPACK;
 		} else if (value.contains("mlp")) {
 			format = FormatConfiguration.MLP;
+		} else if (value.equals("openmg")) {
+			format = FormatConfiguration.ATRAC;
 		} else if (value.contains("atrac")) {
 			format = FormatConfiguration.ATRAC;
-			if (media.getCodecV() == null && !media.getContainer().equals(FormatConfiguration.RM)) {
-				media.setContainer(FormatConfiguration.ATRAC);
-			}
 		} else if (value.equals("jpeg")) {
```

Comment:
@Sami32 The above code isn't mine at all, I just added "RM": https://github.com/UniversalMediaServer/UniversalMediaServer/commit/81680c3f398a3d3fd032e1987f92a011c2ffc063#diff-6ecbf84dbeada135aeff5865473cf43eR330

What this does is to set the *container* to ATRAC if the *audio codec* is ATRAC. That breaks video files with ATRAC audio (if that exists) and it breaks the RM files. I don't remember if there's a "pure" ATRAC file format, but if it is removing this will no longer set their container correcly.